src/java.desktop/share/classes/javax/swing/RepaintManager.java

```diff
@@ -211,22 +208,13 @@ public void removeRepaintListener(RepaintManager rm, RepaintListener l) {
             }
         });
 
-        @SuppressWarnings("removal")
-        var t1 = "true".equals(AccessController.
-                doPrivileged(new GetPropertyAction(
```

Comment:
Missed the GetPropertyAction import for this file (can't annotate there)

Reply:
This will need to be a follow-up at this point.

Reply:
oops. I was already integrating when this comment came in.
I will fix in a follow-up.